src/components/dashboard/FaceRecognition/FaceRecognition.js

```diff
@@ -41,106 +36,60 @@ type State = {
 class FaceRecognition extends React.Component<FaceRecognitionProps, State> {
   state = {
     showPreText: false,
-    showZoomCapture: true,
+    showCamera: true,
     showGuidedFR: false,
     sessionId: undefined,
     loadingText: '',
     facemap: new Blob([], { type: 'text/plain' }),
     zoomReady: false,
-    captureResult: {},
     isWhitelisted: undefined,
     showHelper: get(this.props, 'screenProps.screenState.showHelper', true),
   }
 
-  loadedZoom: any
-
   timeout: TimeoutID
 
-  containerRef = createRef()
-
   width = 720
 
   height = 0
 
-  componentWillUnmount = () => {
-    log.debug('Unloading ZoomSDK', this.loadedZoom)
-    this.timeout && clearTimeout(this.timeout)
-  }
-
-  componentWillMount = async () => {
-    await zoomSdkLoader.ready
-    this.loadedZoom = ZoomSDK
-
-    navigator.getMedia =
-      navigator.getUserMedia || // use the proper vendor prefix
-      navigator.webkitGetUserMedia ||
-      navigator.mozGetUserMedia ||
-      navigator.msGetUserMedia
-
-    navigator.getMedia(
-      { video: true },
-      () =>
-        (this.timeout = setTimeout(() => {
-          this.setState({ zoomReady: true })
-        }, 0)),
-      this.showFRError
-    )
-  }
-
-  componentDidMount = () => {}
-
-  /**
-   *  unused
-   */
-  setWidth = () => {
-    const containerWidth =
-      (this.containerRef && this.containerRef.current && this.containerRef.current.offsetWidth) || 300
-    this.width = Math.min(this.width, containerWidth)
-    this.height = window.innerHeight > window.innerWidth ? this.width * 1.77777778 : this.width * 0.5625
-    log.debug({ containerWidth, width: this.width, height: this.height })
-  }
-
-  onCaptureResult = (captureResult?: ZoomCaptureResult): void => {
-    //TODO: rami check uninitilized, return
-    log.debug('zoom capture completed', { captureResult })
+  onCaptureResult = (face, images): void => {
     fireEvent('FR_Capture')
-    if (captureResult === undefined) {
+    if (images === undefined || images.length === 0) {
       log.error('empty capture result')
```

Comment:
match all log.error to our error logging convention

Reply:
log.error(e.message,e,{extra variables})

Reply:
@StanislavShevchenko @serdiukov-o-nordwhale

Reply:
@sirpy thanks, now i'll know about this

Reply:
done